app/Services/Caches/CampaignCacheService.php

```diff
@@ -245,6 +245,9 @@ public function themes(): string|bool
         /** @var CampaignPlugin $plugin */
         foreach ($plugins as $plugin) {
             $theme .= "/** plugin: " . e($plugin->name) . " #" . e($plugin->version->version) . " **/\n";
+            $theme .= $plugin->version->fonts . "\n\n";
+        }
+        foreach ($plugins as $plugin) {
             $theme .= $plugin->version->content . "\n\n";
```

Comment:
Would be great to have a comment in the css file again to quickly find declarations for debugging